lib/diagnostic.js

```diff
@@ -8,22 +8,35 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  let result = arguments ;
+  for (let i = 0; i < arguments.length; i++) {
+    result = arguments[i];
+  }
+return result;
```

Comment:
This function and ```sum()``` will only return the argument if there is one argument, and the last argument if there is more than one argument.